docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -179,25 +180,23 @@ yield return Environment.NewLine;
 ```
 
 Next, you need to modify how you consume the lines of the file, and add a
-delay after writing each word. Replace the `Console.WriteLine()` statement
+delay after writing each word. Replace the `Console.WriteLine(line)` statement
 in the `Main` method with the following block:
 
 ```cs
+Console.Write(line);
+if (!string.IsNullOrWhiteSpace(line))
 {
-    Console.Write(line);
-    if (!string.IsNullOrWhiteSpace(line))
-    {
-        var pause = Task.Delay(200);
-        // Synchronously waiting on a task is an
-        // anti-pattern. This will get fixed in later
-        // steps.
-        pause.Wait();
-    }
+    var pause = Task.Delay(200);
+    // Synchronously waiting on a task is an
+    // anti-pattern. This will get fixed in later
+    // steps.
+    pause.Wait();
```

Comment:
Rather then create a task and then wait for its completion (and include an awkward comment), it might be easier to just call Thread.Sleep(200)